retromock/src/main/java/co/infinum/retromock/KotlinExtensions.kt

```diff
@@ -0,0 +1,98 @@
+@file:JvmName("KotlinExtensions")
+
+package co.infinum.retromock
+
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.suspendCancellableCoroutine
+import retrofit2.*
+import kotlin.coroutines.resumeWithException
+import kotlin.coroutines.intrinsics.COROUTINE_SUSPENDED
+import kotlin.coroutines.intrinsics.intercepted
+import kotlin.coroutines.intrinsics.suspendCoroutineUninterceptedOrReturn
+import kotlin.coroutines.resume
+
+inline fun <reified T> Retrofit.create(): T = create(T::class.java)
+
+suspend fun <T : Any> Call<T>.await(): T {
+    return suspendCancellableCoroutine { continuation ->
+        continuation.invokeOnCancellation {
+            cancel()
+        }
+        enqueue(object : Callback<T> {
+            override fun onResponse(call: Call<T>, response: Response<T>) {
+                if (response.isSuccessful) {
+                    val body = response.body()
+                    if (body == null) {
+                        val e = KotlinNullPointerException("Response body type was declared as non-null")
+                        continuation.resumeWithException(e)
+                    } else {
+                        continuation.resume(body)
+                    }
+                } else {
+                    continuation.resumeWithException(HttpException(response))
+                }
+            }
+
+            override fun onFailure(call: Call<T>, t: Throwable) {
+                continuation.resumeWithException(t)
+            }
+        })
+    }
+}
+
+@JvmName("awaitNullable")
+suspend fun <T : Any> Call<T?>.await(): T? {
```

Comment:
Why this function isn't used?

Reply:
It seems we didn't need it. I'll remove it.